optimization:
  radius: 0.2
  max_steps: 0
  step_improvement_threshold: 0.01
  max_threads: 1
    #  seed_state:
    #    - name: elbow_joint
    #      position: 1.74
    #    - name: shoulder_lift_joint
    #      position: -1.57
    #    - name: shoulder_pan_joint
    #      position: 0.0
    #    - name: wrist_1_joint
    #      position: -3.29
    #    - name: wrist_2_joint
    #      position: -1.57
    #    - name: wrist_3_joint
    #      position: 0.0
    #    - name: endo_first_joint
    #      position: 0.0
    #    - name: endo_second_joint_first_dof
    #      position: 0.0
    #    - name: endo_second_joint_second_dof
    #      position: 0.0

ik_solver:
  name: MoveItIKSolver
  distance_threshold: 0.0
  planning_group: all
  collision_mesh_filename: package://benchmark/config/hydrogen.ply
  touch_links: [endo_third_link, end_effector]

evaluator:
  name: MultiplicativeEvaluator
  plugins:
    - name: ManipulabilityMoveIt
      planning_group: all
    - name: DistancePenaltyMoveIt
      planning_group: all
      distance_threshold: 0.025
      exponent: 2
      collision_mesh_filename: package://benchmark/config/hydrogen.ply
      touch_links: [endo_third_link, end_effector]

display:
  name: ROSDisplay
  collision_mesh_filename: package://benchmark/config/hydrogen.ply
  kinematic_base_frame: base_link
  marker_scale: 0.025

target_pose_generator:
  name: PointCloudTargetPoseGenerator
  pcd_file: package://benchmark/config/hydrogen_single.pcd

logger:
  name: BoostProgressConsoleLogger
